src/ingest_validation_tools/directory-schemas/lightsheet.yaml

```
@@ -1,3 +1,72 @@
+# Level0/
 -
-  pattern: 'TODO'
-  description: 'Directory structure not yet specified.'
+  pattern: 'Level0/Channel+[^/]/[^/]+\.csv'
```

```suggestion
  pattern: 'Level0/Channel[^/]+/[^/]+\.csv'
```

What you have allows the `l` to be repeated.